index.d.ts

```diff
@@ -0,0 +1,12 @@
+import { Parser } from 'acorn' 
+
+declare const jsx: (options?: jsx.AcornJSXOptions) => (BaseParser: typeof Parser) => typeof Parser;
+
+declare namespace jsx {
+  interface AcornJSXOptions {
```

Comment:
```suggestion
  interface Options {
```

It's already namespaced under the module, I'd just use `Options` directly.

Reply:
Since more packages define types named `Options`, this often causes naming conflicts. I think it’s best to keep the original name. Otherwise, this may result in code such as:

```ts
import { Options as AcornOptions } from 'acorn';
import { Options as AcornJSXOptions } from 'acorn';
```

Reply:
I think that's fine - it's exactly what `as` exists for. What's the point of having real modules if we still need to manually name's pace things on the library side :)

Reply:
Alright, let’s agree to disagree then. This is subjective and it’s not my library, so I’ll apply the suggestion. :)